darkmoon/app.py

```diff
@@ -4,12 +4,24 @@
 # Proprietary and confidential.
 
 
-from fastapi import FastAPI
+from fastapi import FastAPI, Request
+from fastapi.responses import HTMLResponse, Response
+from fastapi.templating import Jinja2Templates
 
 import darkmoon.api.v1.metadata.views as views
 from darkmoon.core.database import register_database
 from darkmoon.settings import Settings
 
+app = FastAPI()
```

Comment:
FastAPI app creation already happens later in this file, please remove this line